integration_tests/header.sh

```diff
@@ -3,7 +3,6 @@
 my_dir="$(dirname "$0")"
 
 #set -o errexit ... re-enable when we fix the storage deal bug on Lotus side
-set -x
```

Comment:
don't we want this?

Reply:
See the last part of the commit message.